scripts/authors/index.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+
+const { promises: { writeFile } } = require('fs')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/omrilotan/isbot/pullRequest?prid=7728710)
```suggestion
const { promises: { writeFile } } = require("fs")
```